core/server/worker/src/main/java/alluxio/worker/grpc/BlockReadHandler.java

```diff
@@ -90,6 +90,9 @@
   private static final Logger SLOW_BUFFER_LOG = new SamplingLogger(LOG, Constants.MINUTE_MS);
   private static final long SLOW_BUFFER_MS =
       ServerConfiguration.getMs(PropertyKey.WORKER_REMOTE_IO_SLOW_THRESHOLD);
+  /** Metrics */
+  private static final Counter RPC_READ_COUNT =
+      MetricsSystem.counter(MetricKey.WORKER_RPC_READ_COUNT.getName());
```

Comment:
Is this the correct way to 'register' a metric? When I compiled and tested I didn't see this get reported in the `/metrics/json` sink.

Reply:
did you go to the `master/metrics/json` or the `worker/metrics/json`

Reply:
Apparently the metric wasn't appearing because I didn't trigger the `Counter.inc()` calls in the code during my test.

In a subsequent test (read a file from S3 bucket UFS, not cached in Alluxio FS), I saw the `Worker.RpcReadCount` metric appear at the `<worker>:30000/metrics/json` endpoint